test/kino_slack/messsage_interpolator_test.exs

```diff
@@ -0,0 +1,45 @@
+defmodule KinoSlack.MesssageInterpolatorTest do
+  use ExUnit.Case, async: true
+
+  alias KinoSlack.MessageInterpolator, as: Interpolator
+
+  test "it interpolates variables inside a message" do
+    first_name = "Hugo"
+    last_name = "Baraúna"
+    message = "Hi {{first_name}} {{last_name}}! 🎉"
+
+    interpolated_ast = Interpolator.interpolate(message)
+    {interpolated_message, _} = Code.eval_quoted(interpolated_ast, binding())
+
+    assert interpolated_message == "Hi Hugo Baraúna! 🎉"
+  end
+
+  test "it interpolates expressons inside a message" do
+    message = "One plus one is: {{1 + 1}}"
+
+    interpolated_ast = Interpolator.interpolate(message)
```

Comment:
I would change the tests so we also check on the result of `Macro.to_string(interpolated_ast)`. This is because the ast we emit is public (it will be visible by users), so we want to make sure it looks nice.

Reply:
Done. ✅